presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -40,6 +40,7 @@
 import java.util.stream.Collectors;
```

Comment:
Haha in this case I would say the commit message is *too* detailed. You don't need to say what it's *not* about.